src/components/partnership/partnership.service.ts

```diff
@@ -241,6 +244,25 @@ export class PartnershipService {
         })
         .first();
 
+      const fiscalRange = fiscalYears(input.mouStart, input.mouEnd);
```

Comment:
please add some comments to explain what and why this code is doing.

Reply:
this is old code that's been in there the whole time.  It calculates which fiscal years are represented by the mouStart to mouEnd.  IE Dec 1 2020 is in FY 2021

Reply:
added comment